tests/e2e/fixtures/kserve_dependencies.py

```diff
@@ -0,0 +1,204 @@
+import os
+import logging
+import re
+import subprocess
+import time
+import pytest
+from e2e.utils.constants import TENSORFLOW_SERVING_VERSION
+from e2e.utils.s3_for_training.data_bucket import S3BucketWithTrainingData
+from e2e.fixtures.cluster import (
+    cluster,
+    create_iam_service_account,
+    associate_iam_oidc_provider,
+    delete_iam_service_account,
+)
+from e2e.utils.utils import (
+    rand_name,
+    write_yaml_file,
+    exec_shell,
+    wait_for,
+    load_yaml_file,
+    kubectl_apply,
+)
+from e2e.utils.config import configure_resource_fixture, metadata
+from e2e.conftest import region
+from e2e.utils.load_balancer.setup_load_balancer import create_certificates
+from e2e.utils.custom_resources import get_inference_service
+
+
+RANDOM_PREFIX = rand_name("kserve-")
+SECRET_CONFIG_FILE = "./resources/kserve/kserve-secret.yaml"
+INFERENCE_CONFIG_FILE = "./resources/kserve/inference-service.yaml"
+AUTHORIZATION_POLICY_CONFIG_FILE = "./utils/kserve/allow-predictor-transformer.yaml"
+PROFILE_NAMESPACE = "kubeflow-user-example-com"
```

Comment:
Can use this https://github.com/awslabs/kubeflow-manifests/blob/9fcd434257fef6d5bfa1667647cd67fae53cb540/tests/e2e/utils/constants.py#L6